src/screens/TopicPageScreen/index.js

```diff
@@ -287,14 +287,15 @@ const TopicPageScreen = (props) => {
   };
 
   const onPress = (item, haveSeeMore) => {
-    props.navigation.navigate('PostDetailPage', {
-      feedId: item.id,
+    const data = {
+      feedId: item?.id,
       isalreadypolling: item.isalreadypolling,
       from: 'topic',
       isCaching: true,
       haveSeeMore,
       data: item
-    });
+    };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `data` object is created with properties that are directly accessed from the `item` object. This could potentially lead to undefined errors if the `item` object does not have these properties. It would be safer to use optional chaining when accessing these properties.
```diff
-      feedId: item?.id,
-      isalreadypolling: item.isalreadypolling,
-      data: item
+      feedId: item?.id ?? null,
+      isalreadypolling: item?.isalreadypolling ?? false,
+      data: item ?? {}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->